README.md

```diff
@@ -3,14 +3,19 @@
 [![pod](https://badge.fury.io/co/Iterable-iOS-SDK.svg)](https://cocoapods.org/pods/Iterable-iOS-SDK)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 
-# Table of contents
+# Iterable iOS SDK
+
+The Iterable iOS SDK is a Swift implementation of an iOS client for Iterable, for iOS versions 9.0 and higher.
+
+## Table of contents
 
+- [Before starting](#before-starting)
 - [Installation](#installation)
-- [Installing with a dependency manager](#installing-with-a-dependency-manager)
     - [Swift Package Manager](#swift-package-manager)
     - [CocoaPods](#cocoapods)
     - [Carthage](#carthage)
     - [Manual Installation](#manual-installation)
+    - [Beta versions](#beta-versions)
```

Comment:
Do we need beta version information?

Reply:
Closing this out as we discussed on Slack; going to keep this here for now.